tests/testthat/_snaps/notes.md

```diff
@@ -79,3 +79,14 @@
     Output
       Great job! No notes to show.
 
+---
+
+    Code
+      show_notes(fit_lr)
+    Output
+      unique notes:
+      -----------------------------------
+      glm.fit: algorithm did not converge 
+       -----------------------------------
```

Comment:
This might be a nitpick, but why does this line start with a space? 😅

Reply:
Lacking a `sep = ""` in `cat()`